docs/src/app/(docs)/api-reference/client/page.mdx

```diff
@@ -0,0 +1,265 @@
+import { docsMetadata } from '@/lib/utils'
+
+export const metadata = docsMetadata({
+  title: 'uploadthing/client',
+  description:
+    'The UploadThing Client module provides utilities for working files in your application and communicating with your backend file router.',
+  category: 'API Reference',
+})
+
+# uploadthing/client
+
+The UploadThing Client module provides utilities for working files in your application
+and communicating with your backend file router.
+
+## `uploadFiles` {{ tag: 'function', since: '5.0' }}
```

Comment:
thought: we should be able to do like `{{ since: 'UNRELEASED' }}` and have a little script that runs on releases and replaces that with the released version